client/me/privacy/main.jsx

```diff
@@ -0,0 +1,41 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import createReactClass from 'create-react-class';
+import { localize } from 'i18n-calypso';
+import { flowRight as compose } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { protectForm } from 'lib/protect-form';
+import formBase from 'me/form-base';
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+
+// NOTE: disabling it since we need to use the `formBase` mixin
+// eslint-disable-next-line react/prefer-es6-class
+const Privacy = createReactClass( {
+	displayName: 'Privacy',
```

Comment:
Let's make this a standard React component. 

Also, setting the `displayName` isn't necessary here, since we use the `transform-react-display-name` babel transform. (One exception is for example components in DevDocs since we'd like the displayName to not be shortened by the minifier.)